src/order.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default.html
+---
+
+<h2>Order</h2>
+<p>Please use this to order biscuits!</p>
+<form action="/action_page.php">
+    <fieldset>
+      <legend>Order form:</legend>
```

Comment:
`fieldset` is used for combining form elements (like groups of checkboxes), so using it to wrap around a whole form is a bit of overkill; the `form` element is already doing that work.